test/unit/StablePair.t.sol

```diff
@@ -924,7 +924,7 @@ contract StablePairTest is BaseTest {
         bytes32 lEncoded = bytes32(abi.encodePacked(lLastInvariantAmp, lLastInvariant));
         // hardcoding the slot for now as there is no way to access it publicly
         // this will break when we change the storage layout
-        vm.store(address(_stablePair), bytes32(uint256(65_553)), lEncoded);
+        vm.store(address(_stablePair), bytes32(uint256(18)), lEncoded);
```

Comment:
had to change the storage slot as we're no longer using the array for `observations`